### PR TITLE
Adding Arbitrum Chain to CoW Protocol 

### PR DESCRIPTION
**Pull request by olgafetisova**
### Contribution type

- [x] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

### For new spell(s)

- **Spell name(s):** cow_protocol_arbitrum.trades

- **Description:** The spell adds all CoW Protocol settled trades on Arbitrum chain. The spell is designed for the entire community and the data will be included to the dex_aggregator.trades community table.

- **Test instructions:** P
OC query can be found here https://dune.com/queries/3783635

---

### For adding to existing spell lineage

- **Description:** Adding new union to the existing spell with Arbitrum chain to the cow_protocol.trades. The table is CoW Protocol trades on all chains across all contracts and versions. This table loads dex trades downstream.

### CODE REVIEW

**Comment by olgafetisova** [comment]
@jeff-dude would appreciate a review

File: dbt_project.yml [hunk 1]
```diff
@@ -674,6 +674,8 @@ models:
         +schema: cow_protocol_ethereum
       gnosis:
         +schema: cow_protocol_gnosis
+      arbitrum:
+        +schema: cow_protocol_arbitrum
```
Comment (jeff-dude) [inline dbt_project.yml:678]: we no longer needs edit to this file, we're working through cleaning out all schemas from here and applying directly in models instead for easier readability. plz do remove here 🙏

File: models/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql [hunk 2]
```diff
@@ -0,0 +1,225 @@
+{{  config(
+
```
Comment (jeff-dude) [inline models/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql:2]: with the above comment on schemas, plz add here instead

**Review by jeff-dude — CHANGES_REQUESTED** [review]
minor tweaks below

File: models/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql (hunk 2 above)
Comment (olgafetisova) [reply models/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql]: Makes sense, adjusted

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**